source/features/mark-merge-commits-in-list.tsx

```diff
@@ -0,0 +1,56 @@
+import './mark-merge-commits-in-list.css';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+
+interface CommitInfo {
+	parents: {
+		totalCount: number | string;
+	};
+}
+
+const getCommitParentCount = async (commits: string[]): Promise<Record<string, CommitInfo>> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+					... on Commit {
+							parents{
+								totalCount
+							}
+						}
+					}
+			`).join('\n')}
+		}
+	`);
+
+	return repository;
+};
+
+function getCommitHash(commit: HTMLElement): string {
+	return (commit.dataset.channel as string).split(':')[3];
+}
+
+async function init(): Promise<void | false> {
+	const pageCommits = select.all('li.commit');
+	const parentCommitCount = await getCommitParentCount(pageCommits.map(getCommitHash));
+	for (const commit of pageCommits) {
+		const commitHash = getCommitHash(commit);
+		const {totalCount} = parentCommitCount[api.escapeKey(commitHash)].parents;
+		if (totalCount === 2) {
+			commit.classList.add('refined-github-merge-commit');
```

Comment:
Our classes start with `rgh-`

Reply:
@fregante np I just revered the removal commit and started from there. The classes are exactly how they were before.